src/stores/connectors/keys/create.go

```diff
@@ -0,0 +1,24 @@
+package keys
+
+import (
+	"context"
+	"github.com/consensys/quorum-key-manager/src/stores/store/entities"
+)
+
+func (c Connector) Create(ctx context.Context, id string, alg *entities.Algorithm, attr *entities.Attributes) (*entities.Key, error) {
+	logger := c.logger.With("id", id).With("algorithm", alg.Type).With("curve", alg.EllipticCurve)
+	logger.Debug("creating key")
+
+	key, err := c.store.Create(ctx, id, alg, attr)
+	if err != nil {
+		return nil, err
+	}
+
+	err = c.db.Keys().Add(ctx, key)
+	if err != nil {
+		return nil, err
```

Comment:
Shouldn't we "UnCreate" the created key in that case ?

Reply:
to discuss

Reply:
won't have time to discuss this so I created a ticket: https://app.zenhub.com/workspaces/orchestrate-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/242